prow/test/integration/config/prow/cluster/webhook_server_deployment.yaml

```diff
@@ -67,3 +70,5 @@ spec:
       - name: config
         configMap:
           name: config
+      - name: prowjob-webhook-ca-cert
+        emptyDir: {}
```

Comment:
Discussed offline, we need to use a persistent volume mount if we want to have an integration test for availability during pod restarts/upgrades.